presto-main/src/main/java/com/facebook/presto/FullConnectorSession.java

```diff
@@ -166,7 +184,7 @@ public String toString()
                 .add("traceToken", getTraceToken().orElse(null))
                 .add("locale", getLocale())
                 .add("startTime", getStartTime())
-                .add("properties", properties)
+                .add("metastoreHeaders", getMetastoreHeaders())
```

Comment:
Do we need this change ?

Reply:
No we don't I can remove, just wanted to add it for completeness